packages/wallet-frontend/src/contexts/Application.jsx

```diff
@@ -35,6 +35,9 @@ export default function Provider({ children }) {
       if (type === 'walletUpdateInbox') {
         dispatch(updateInbox(JSON.parse(data)));
       }
+      if (type === 'walletAcceptOfferResponse') {
```

Comment:
@Chris-Hibbert I think I'm missing something or there was a weird merge. Can you rebase on master? this is already in master but it's showing up as a change.

https://github.com/Agoric/agoric-sdk/blob/7fa8c7969bb2e8fffd661e28e227c37de031b32f/packages/wallet-frontend/src/contexts/Application.jsx#L38

```
 if (type === 'walletAcceptOfferResponse') {
        alert(data.outcome);
      }
```